core-service/src/domain/mocks/repositories/PopUpBannerRepositoryMocks.go

```diff
@@ -0,0 +1,60 @@
+// Code generated by mockery v2.14.1. DO NOT EDIT.
+
+package mocks
+
+import (
+	context "context"
+
+	domain "github.com/jabardigitalservice/portal-jabar-services/core-service/src/domain"
+	mock "github.com/stretchr/testify/mock"
+)
+
+// PopUpBannerRepository is an autogenerated mock type for the PopUpBannerRepository type
+type PopUpBannerRepository struct {
+	mock.Mock
+}
+
+// Fetch provides a mock function with given fields: ctx, params
+func (_m *PopUpBannerRepository) Fetch(ctx context.Context, params *domain.Request) ([]domain.PopUpBanner, int64, error) {
+	ret := _m.Called(ctx, params)
+
+	var r0 []domain.PopUpBanner
+	if rf, ok := ret.Get(0).(func(context.Context, *domain.Request) []domain.PopUpBanner); ok {
+		r0 = rf(ctx, params)
+	} else {
+		if ret.Get(0) != nil {
+			r0 = ret.Get(0).([]domain.PopUpBanner)
+		}
+	}
```

Comment:
```suggestion
	if ret.Get(0) != nil {
		r0 = ret.Get(0).([]domain.PopUpBanner)
	}
	if rf, ok := ret.Get(0).(func(context.Context, *domain.Request) []domain.PopUpBanner); ok {
		r0 = rf(ctx, params)
	}
```

what happen when we change current code to this suggested?

Reply:
hii Kang @rindibudiaramdhan by the way this code is not made by humans. All script files on `/mocks/...` was generated by [mockery](https://github.com/vektra/mockery) instead, so we don't have permission to edit the file generated. 

FYI, mockery provides the ability to easily generate mocks for Golang interfaces. 😃

Reply:
oh okay. need send this issue to mockery instead then LOL